docs/src/main/asciidoc/security-basic-authentication-howto.adoc

```diff
@@ -16,7 +16,11 @@ Enable Basic authentication for your Quarkus project and allow users to authenti
 quarkus.http.auth.basic=true
 ----
  
-For a Basic authentication configuration walk-through that uses `JPA`, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication] guide.
+The easiest way to configure user credentials required for a Basic authentication to work is to configure the user name, secret and roles directly in `application.properties`, see the xref:security-testing.adoc#configuring-user-information[Configuring User Information] section for more information.
```

Comment:
@sberyozkin - 2 things:

1. Sentence contains 35 words.
The above-suggested change reduces the sentence length. To improve readability and clarity, the max sentence length recommendation for Quarkus contributors (defined in doc-reference.adoc) is 32 words, with the exception of the first sentence (preamble), which must be less than 26 to fit/look good on the index page tile.
[When the Vale linter is fully implemented, it will detect and flag this btw...]. 

2. References: We try to use a consistent approach to different types of cross-references across all of the docs. The above is widely used across the product docs and will support single-sourcing too. We can make the contributor guide a bit clearer here and add examples. I can take that action ...

Reply:
Thanks for the explanation @michelle-purcell